docs/4.3/admin-guide.md

```diff
@@ -1565,8 +1565,10 @@ Teleport will handle its own SSL on top of that with its own certificates.
     balancer you'll need to run Teleport with `--insecure-no-tls`
 
 If your load balancer supports HTTP health checks, configure it to hit the
-`/webapi/ping` endpoint on the proxy. This endpoint will reply `200 OK` if the
-proxy is running without problems.
+`/readyz` [diagnostics endpoint](metrics_logs_reference.md) on the proxy. This endpoint
```

Comment:
Technically you can use this endpoint for healthchecks of any Teleport service (auth, proxy or node) so maybe we change the language a bit for that?